README.md

```diff
@@ -92,158 +76,166 @@ Additionally, the device and any additional complementary hardware components sh
-
-
-### Board specific examples
-
-By board vendor alphabetic order.
-
-Platform                        |  Connectivity     | Storage   | URL to example
---------------------------------| ----------------- | --------- | --------------------
-Nuvoton NUMAKER-IOT-M487        | WiFi              | SD Card   | https://os.mbed.com/teams/Nuvoton/code/pelion-example-common/
-Nuvoton NUMAKER-PFM-M487        | Ethernet          | SD Card   | https://os.mbed.com/teams/Nuvoton/code/pelion-example-common/
-Nuvoton NUMAKER-PFM-NUC472      | Ethernet          | SD Card   |https://os.mbed.com/teams/Nuvoton/code/pelion-example-common/
-NXP FRDM-K64F                   | Ethernet          | SD Card   | https://os.mbed.com/teams/NXP/code/mbed-cloud-connect-example-ethernet
-NXP FRDM-K66F                   | Ethernet          | SD Card   | https://os.mbed.com/teams/NXP/code/mbed-cloud-connect-example-ethernet
-Renesas GR-LCYHEE               | WiFi              | SD Card   | https://os.mbed.com/teams/Renesas/code/pelion-example-common/
-Sigma Delta Technologies SDT64B | Ethernet          | SD Card   | https://os.mbed.com/teams/Sigma-Delta-Technologies/code/pelion-example-common
-ST DISCO_L475E_IOT01A           | WiFi              | QSPI      | https://os.mbed.com/teams/ST/code/pelion-example-disco-iot01/
-ST DISCO_F413H                  | WiFi              | QSPI      | https://os.mbed.com/teams/ST/code/pelion-example-common/
-ST DISCO_F746NG                 | Ethernet          | QSPI      | https://os.mbed.com/teams/ST/code/pelion-example-common/
-ST NUCLEO_F429ZI                | Ethernet          | SD Card   | https://os.mbed.com/teams/ST/code/pelion-example-common/
-ST NUCLEO_F767ZI                | Ethernet          | SD Card   | https://os.mbed.com/teams/ST/code/pelion-example-common/
-ST NUCLEO_F746ZG                | Ethernet          | SD Card   | https://os.mbed.com/teams/ST/code/pelion-example-common/
-ST NUCLEO_F207ZG                | Ethernet          | SD Card   | https://os.mbed.com/teams/ST/code/pelion-example-common/
-UBlox EVK ODIN W2               | WiFi              | SD Card   | https://os.mbed.com/teams/ublox/code/pelion-example-common/
-UBlox C030 U201                 | Cellular          | SD Card   | https://os.mbed.com/teams/ublox/code/pelion-example-common/
-
+To help you start quickly, please refer to the following [application example](https://github.com/ARMmbed/pelion-ready-example). It demonstrates how to connect to the Pelion IoT Platform service, register resources and get ready to receive a firmware update.
+
+Also, there are a number of board-specific applications that focus on providing more elaborate hardware features with Mbed OS and the Pelion IoT Platform. These are available in the Pelion [quick start](https://cloud.mbed.com/quick-start). Please see the reference table below, organized by vendor name, for details:
+
+Platform                        |  Connectivity      | Storage   | Example URL
```

Comment:
Will this table need to be updated frequently? Will we be adding to it each release?

Reply:
It will get updated per minor release